plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveStorageFormat.java

```diff
@@ -61,57 +61,70 @@ public enum HiveStorageFormat
     ORC(
             ORC_SERDE_CLASS,
             ORC_INPUT_FORMAT_CLASS,
-            ORC_OUTPUT_FORMAT_CLASS),
+            ORC_OUTPUT_FORMAT_CLASS,
+            ".orc"),
     PARQUET(
             PARQUET_HIVE_SERDE_CLASS,
             MAPRED_PARQUET_INPUT_FORMAT_CLASS,
-            MAPRED_PARQUET_OUTPUT_FORMAT_CLASS),
+            MAPRED_PARQUET_OUTPUT_FORMAT_CLASS,
+            ".parquet"),
     AVRO(
             AVRO_SERDE_CLASS,
             AVRO_CONTAINER_INPUT_FORMAT_CLASS,
-            AVRO_CONTAINER_OUTPUT_FORMAT_CLASS),
+            AVRO_CONTAINER_OUTPUT_FORMAT_CLASS,
+            ".avro"),
     RCBINARY(
             LAZY_BINARY_COLUMNAR_SERDE_CLASS,
             RCFILE_INPUT_FORMAT_CLASS,
-            RCFILE_OUTPUT_FORMAT_CLASS),
+            RCFILE_OUTPUT_FORMAT_CLASS,
+            ".rc"),
     RCTEXT(
             COLUMNAR_SERDE_CLASS,
             RCFILE_INPUT_FORMAT_CLASS,
-            RCFILE_OUTPUT_FORMAT_CLASS),
+            RCFILE_OUTPUT_FORMAT_CLASS,
+            ".rc"),
     SEQUENCEFILE(
             LAZY_SIMPLE_SERDE_CLASS,
             SEQUENCEFILE_INPUT_FORMAT_CLASS,
-            HIVE_SEQUENCEFILE_OUTPUT_FORMAT_CLASS),
+            HIVE_SEQUENCEFILE_OUTPUT_FORMAT_CLASS,
+            ".seq"),
     JSON(
             JSON_SERDE_CLASS,
             TEXT_INPUT_FORMAT_CLASS,
-            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS),
+            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS,
+            ".json"),
     OPENX_JSON(
             OPENX_JSON_SERDE_CLASS,
             TEXT_INPUT_FORMAT_CLASS,
-            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS),
+            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS,
+            ".json"),
     TEXTFILE(
             LAZY_SIMPLE_SERDE_CLASS,
             TEXT_INPUT_FORMAT_CLASS,
-            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS),
+            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS,
+            ".txt"),
     CSV(
             OPENCSV_SERDE_CLASS,
             TEXT_INPUT_FORMAT_CLASS,
-            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS),
+            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS,
+            ".csv"),
     REGEX(
             REGEX_SERDE_CLASS,
             TEXT_INPUT_FORMAT_CLASS,
-            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS);
+            HIVE_IGNORE_KEY_OUTPUT_FORMAT_CLASS,
+            "");
```

Comment:
what should the extension be here ?

Reply:
Regex is not a writable format, so it does not need an extension.